nesta/core/batchables/nlp/bert_vectorize/run.py

```diff
@@ -65,15 +65,29 @@ def run():
     # Convert text to vectors
     embeddings = model.encode(docs)
     # Write to output
-    out_data = [{"article_id": _id, "vector": embedding.tolist()}
+    out_data = [{id_field: _id, "vector": ["%.5f" % v for v in embedding.tolist()]}
```

Comment:
Truncating the vectors to 5 decimal places, rather than 20 to save on bytes just for unnecessary (and also illegible) accuracy. This also makes the ingestion a little quicker.